src/dsp_tools/utils/connection_live.py

```diff
@@ -51,19 +51,18 @@ def login(self, email: str, password: str) -> None:
             password: password of the user
 
         Raises:
-            BaseError: if DSP-API returns no token with the provided user credentials
+            PermanentConnectionError: if DSP-API returns no token with the provided user credentials
         """
-        response = self.post(
-            route="/v2/authentication",
-            data={"email": email, "password": password},
-        )
-        if not response.get("token"):
-            raise BaseError(
-                f"Error when trying to login with user '{email}' and password '{password} "
-                f"on server '{self.server}'",
-                json_content_of_api_response=json.dumps(response),
-                api_route="/v2/authentication",
+        err_msg = f"Username and/or password are not valid on server '{self.server}'"
+        try:
+            response = self.post(
+                route="/v2/authentication",
+                data={"email": email, "password": password},
             )
+        except PermanentConnectionError:
+            raise UserError(err_msg)
+        if not response.get("token"):
+            raise UserError(err_msg)
```

Comment:
@Nora-Olivia-Ammann With this, the user gets a nice feedback, but no sensitive data is logged.